cpp/splinepy/py/py_spline.hpp

```diff
@@ -928,7 +990,8 @@ inline void add_spline_pyclass(py::module& m, const char* class_name) {
                              &splinepy::py::PySpline::ParametricBounds)
       .def_property_readonly("control_mesh_resolutions",
                              &splinepy::py::PySpline::ControlMeshResolutions)
-
+      .def_property_readonly("evaluate_boundary_centers",
```

Comment:
```suggestion
      .def("evaluate_boundary_centers",
```
If you wanna keep the name `evaluate_*`, make this a method instead of property

Reply:
Cut the `evaluate_` part